src/GuppyDataExplorer/ExplorerFilterSet/utils.js

```diff
@@ -4,26 +4,6 @@ import './typedef';
 
 const FILTER_SET_URL = '/amanuensis/filter-set';
 
```

Comment:
Shouldn't line 5 reference 'filter-sets' as in FILTER_SET_URL = '/amanuensis/filter-sets' ?

Reply:
Thanks for pointing this out. I'm aware of the URL change, but that had to wait since I was already well into it and testing against the old amanuensis Docker image. I.e. I'll be following up with another PR to adopt the new API for filter sets endponint.